foodsaving/groups/serializers.py

```diff
@@ -274,7 +274,10 @@ def update(self, instance, validated_data):
 
 class GroupMembershipAddNotificationTypeSerializer(serializers.Serializer):
     notification_type = serializers.ChoiceField(
-        choices=(GroupNotificationType.WEEKLY_SUMMARY,),
+        choices=(
```

Comment:
Ah, much easier than expected. I guess that also works for model fields: "An iterable (e.g., a list or tuple) consisting itself": https://docs.djangoproject.com/en/2.0/ref/models/fields/